src/page/dashboard/__test__/DashboardPage.test.tsx

```
@@ -0,0 +1,223 @@
+        } as any);
+
+        render(<DashboardPage />);
+        expect(screen.getByTestId('skeleton')).toBeInTheDocument();
+    });
+
+    it('formats and displays credits correctly', () => {
+        vi.mocked(useCredits).mockReturnValue({
+            credits: 1250000,
+            orders: [],
+            isLoading: false
+        } as any);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)